gtk/MessageLogWindow.cc

```diff
@@ -201,11 +202,12 @@ void MessageLogWindow::Impl::level_combo_changed_cb(Gtk::ComboBox* combo_box)
 
 namespace
 {
+using namespace std::chrono;
```

Comment:
Limit the exposure:

```suggestion
using std::chrono::system_clock;
```

OTOH, why here if nowhere else? :)

Reply:
ff5692552c66fd2a4b73df302dad7d81ffab6cb8